database/migrations/2021_06_22_161412_create_feestatements-table.php

```diff
@@ -19,7 +19,7 @@ public function up()
             $table->dateTime('date');
             $table->string('document_number')->nullable();
             $table->string('document_type')->nullable();
-            $table->integer('amount');
+            $table->double('amount');
```

Comment:
This is not how you edit a migration. This migration has already been run. So on the database, it will never run again unless you delete the db and refresh all migrations. This is the correct way to edit a migration: https://laravel.com/docs/8.x/migrations#updating-column-attributes